packages/gasket-cli/test/unit/scaffold/actions/global-prompts.test.js

```diff
@@ -54,40 +55,47 @@ describe('globalPrompts', () => {
     assume(promptStub).is.called(3);
   });
 
+  it('overrides inquirer prompt function with --no-prompts', async () => {
+    promptStub.returns({});
+    mockContext.prompts = false;
+    await globalPrompts(mockContext);
+    assume(mockImports.inquirer.prompt).is.a('function');
+  });
```

Comment:
I'm not sure this test is accurate or necessary.

Reply:
Yeah I was having a hard time figuring out how to test this [line](https://github.com/godaddy/gasket/blob/pfx-109/packages/gasket-cli/src/scaffold/actions/global-prompts.js#L153)